tests/rptest/tests/topic_recovery_test.py

```diff
@@ -332,8 +332,14 @@ def __init__(self, s3_client: S3Client, kafka_tools: KafkaCliTools,
     def create_initial_topics(self):
```

Comment:
i think this should be the last commit of this PR

Reply:
agree